README.md

```diff
@@ -170,3 +170,89 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+# Deployment to Fly.io
+
+## Install FlyCtl
+
+To work with the Fly platform, you first need to install Flyctl, a command line interface that allows you to do everything from creating an account to deploy to Fly.
+  - Linux:
+    ```  
+    curl  -L https://fly.io/install.sh | sh
+    ```
+  - OSX:
+    ```
+    brew install flyctl
+    ```
+  - Windows:
+    ```
+    iwr https://fly.io/install.ps1 -useb
+    ```  
+    Here is the [Installation Guilde](https://fly.io/docs/hands-on/install-flyctl/)
+
+
+Next authenticate with your fly.io account:
+  ```
+  fly auth login
+  ```
+
+- To make sure everything is working well:    
+  ```
+  fly apps list
+  ```
+The output of the above command will be empty table since you have no apps launched yet
+
+
+## Configure the Project
+
+### Environment Variables
+
+We shouldn't store secrets in source code, so utilizing environmental variables is needed.
+- Run the following from the root of your project
+  
+  ```
+  #!/usr/bin/env  python3 set_env.py
+  
```

Comment:
Not valid bash. This goes in the header of the python file and python3 removed and script made executable

Reply:
Again. The first script here is where the developer gives their username or handle. The app gets called audio-discrimination-crowdsource-HANDLE

Then they never have to type the app name again